Applications/TransferLearning/CIFAR_Classification/jni/main.cpp

```diff
@@ -421,10 +421,19 @@ int main(int argc, char *argv[]) {
    * @brief     Neural Network Create & Initialization
    */
   nntrainer::NeuralNetwork NN;
+  int status = ML_ERROR_NONE;
   try {
-    NN.loadFromConfig(config);
-    NN.compile();
-    NN.initialize();
+    status = NN.loadFromConfig(config);
+    if (status != ML_ERROR_NONE)
+      return status;
```

Comment:
This looks strange. there is not going to return if there is error. they just throw the error I guess. Isn't it?

Reply:
NN.loadFromConfig can throw as well as return error. So, added both.